ios/Handlers/EventEmitters/CreatePinEventEmitter.swift

```diff
@@ -0,0 +1,28 @@
+class CreatePinEventEmitter: NSObject {
```

Comment:
any reason why the class conforms to NSObject?

Reply:
it shouldnt, will remove it